### ISSUE TITLE
Controls / Analog Arcade Gun, no moving with inputs

### ISSUE THREAD

**Issue by AngeloScipione** [finalburnneo/FBNeo#1881]
Games tested: Lethal enforcers, Jurasic Park, Terminator II.
Software: Retroarch core, 1.19.1 - FinalBurn Neo (v.1.0.0.03)
Hardware: Xbox 360 controller (PC usb).
Configuration: Device type as Analog Arcade Gun. Setting every direction of left stick with "fakes inputs".

Performance expected: When you are moving stick, the cross is moving. If you aren't moving, the cross is in the center. If you move a bit to any direction a mantain, cross move a bit to this direction and mantain in this position.

Performance saw: Nothing. Always in the center,

Feature expected: If cross work with a Xbox 360's gamepad, it can be used with Dolphin Bar in mode 3 because Retroarch detects IR sensor as a stick (with previous configuration: https://github.com/libretro/retroarch-joypad-autoconfig/pull/1196).

**Comment by barbudreadmon** [comment]
>Setting every direction of left stick with "fakes inputs".

I don't understand this part

**Comment by barbudreadmon** [comment]
Oh, i finally understood, you tried remapping.

You don't need to remap anything, your left stick will work properly right after changing the device type.

It seems i forgot to set the input descriptors for that device type, so `Quick Menu > Controls` is listing the non-working dpad inputs instead of the working analog inputs. I now understand your confusion, sorry about that.

**Comment by barbudreadmon** [comment]
I just pushed a fix for the descriptors, thanks for the report.

**Comment by AngeloScipione** [comment]
I'm late. I have seen you resolved, but this it's a confirmation of remapping:

![imagen](https://github.com/user-attachments/assets/351a8209-0824-4932-a5bc-3752d8cc07be)

**Comment by barbudreadmon** [comment]
Again, you didn't need to remap anything, it worked as soon as you changed device type.

**Comment by AngeloScipione** [comment]
> Again, you didn't need to remap anything, it worked as soon as you changed device type.
Yeah, I know now... I tried to explain my original post. I'm sorry.

After some investigation I have detected something else:

- If I broke (cross no moving) Game Remap File (next point), I must "Remove Game Remap File" or "Reset Input Mapping" to restore Analog Arcade Gun's performance.
- If anything is remapped in the left stick, Remap File is corrupted.
- Also, when I have "Analog to Digital Type" as "Left Analog" or "Left Analog (Forced)", Device Type "Analog Arcade Gun" doesn't work. Otherwise, It works with "none", "Right Analog" and  "Left Analog (Forced)"; but only left stick. If I broke a Game Remap File, it doesn't bother change anything... It's freeze.
